# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

bool headLightsOn
bool breakingLightsOn
bool reverseLightsOn
bool turnLeftLightsOn
bool turnRightLightsOn
bool hazardLightsOn